src/codeflare_sdk/utils/generate_yaml.py

```diff
@@ -585,6 +585,8 @@ def enable_openshift_oauth(user_yaml, cluster_name, namespace):
     ray_headgroup_pod["containers"].append(
         client.ApiClient().sanitize_for_serialization(oauth_sidecar)
     )
+    dashboard_route_spec = route_item.get("generictemplate", {}).get("spec")
+    dashboard_route_spec["tls"]["termination"] = "reencrypt"
```

Comment:
```
dashboard_route_spec = route_item.get("generictemplate", {}).get("spec")
dashboard_route_spec["tls"]["termination"] = "reencrypt"
```
These can be removed as they update the default Dashboard Route which is deleted anyway.
The Oauth Route is already hard coded to be reencrypt [here](url)

Reply:
Thanks, have removed